dashboard/app/controllers/course_offerings_controller.rb

```diff
@@ -22,6 +22,77 @@ def update
     render(status: :not_acceptable, plain: e.message)
   end
 
+  def quick_assign_course_offerings
+    offerings = {}
+
+    offerings[:elementary] = {
+      Course: {
```

Comment:
I realized that I believe the `curriculumType` courses will have is "Course" but the title of the column should be "Courses". I don't want to fix that in this PR but something to watch out for. This goes for all of these curriculum types I think